src/query/storage/cache/redis_cache.go

```diff
@@ -202,7 +243,7 @@ func (cache *RedisCache) Set(
 		entry := KeyEncode(entries[i], prefix)
 		// Encode PromResult into string for Redis to store
 		value, err := resultEncode(values[i])
-		cache.logger.Info("Prom result size", zap.Int("length", len(value)), zap.Int("ts_len", len(values[i].PromResult.Timeseries)))
+		// cache.logger.Info("Prom result size", zap.Int("length", len(value)), zap.Int("ts_len", len(values[i].PromResult.Timeseries)))
```

Comment:
we should not comment the code before merge the pr, if we don't need it we can remove it.

Reply:
removed